### PR TITLE
chore(shorebird_cli): v0.3.0

### PR DESCRIPTION
**Pull request by felangel**
## Description

- chore(shorebird_cli): v0.3.0

## Type of Change

- [ ] ✨ New feature (non-breaking change which adds functionality)
- [ ] 🛠️ Bug fix (non-breaking change which fixes an issue)
- [ ] ❌ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🧹 Code refactor
- [ ] ✅ Build configuration change
- [ ] 📝 Documentation
- [X] 🗑️ Chore

### CODE REVIEW

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -2,6 +2,22 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 0.3.0 (May 25, 2023)
+
+We've just released Shorebird CLI v0.2.3 🎉
```
Comment (bryanoltman) [inline RELEASE_NOTES.md]:
```suggestion
We've just released Shorebird CLI v0.3.0 🎉
```

File: RELEASE_NOTES.md [hunk 2]
```diff
@@ -2,6 +2,22 @@
...
+
+We've just released Shorebird CLI v0.3.0 🎉
+
+✨ The most notable changes are:
+
+- 🐦 Support for Flutter 3.10.2 and Dart 3.0.2
+- 🚨 Breaking Change: `shorebird release` was removed in favor of `shorebird release android` as part of the preparation to support iOS
+- 🩹 Fixed a bug which caused crashed Android API <28
+
+📚 Release notes can be found at https://github.com/shorebirdtech/shorebird/releases/tag/v0.2.3
```
Comment (bryanoltman) [inline RELEASE_NOTES.md]:
```suggestion
📚 Release notes can be found at https://github.com/shorebirdtech/shorebird/releases/tag/v0.3.0
```

**Review by bryanoltman — APPROVED** [review]

**Comment by bryanoltman** [comment]
Did the asset diff change make it in?

**Comment by bryanoltman** [comment]
Or rather, did the aab upload *and* asset diff changes make it in? The asset diff doesn't really work without the aab :)

**Comment by felangel** [comment]
> Or rather, did the aab upload _and_ asset diff changes make it in? The asset diff doesn't really work without the aab :)

Yeah I believe both are included. Let me update the release notes.

**Pull request merged**